eng/installLatestFromReleaseBranch.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+# Function to display usage information
+show_help() {
+    echo "Usage: $0 [--fromMain] [--help]"
+    echo ""
+    echo "Options:"
+    echo "  --fromMain  Use the latest build from main branch instead of release."
```

Comment:
my reading of the above is that 
```suggestion
    echo "  --fromMain  Get the latest build from main branch (default: latest build from the release branch)."
```